rolling-shutter/p2p/p2p.go

```diff
@@ -213,6 +216,17 @@ func createHost(
 			// Attempt to open ports using uPNP for NATed hosts.
 			libp2p.NATPortMap(),
 		)
+		if len(config.BootstrapPeers) > 0 {
+			options = append(options,
+				libp2p.EnableAutoRelayWithStaticRelays(config.BootstrapPeers),
```

Comment:
We could possibly also use the access nodes here, but that would have been more work to gather the addresses.

I'd say lets test with this first and see if it changes anything.